src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -1284,10 +1321,11 @@ void GetUserConvectionCoefficients(EnergyPlusData &state)
                     if (HcExt_ConvectionTypesMap.find(equationName) != HcExt_ConvectionTypesMap.end()) {
                         ExtValue = HcExt_ConvectionTypesMap.at(equationName);
 
-                        if ((equationName == "SIMPLECOMBINED") || (equationName == "TARP") || (equationName == "MOWITT") ||
-                            (equationName == "DOE-2") || (equationName == "ADAPTIVECONVECTIONALGORITHM")) {
+                        if ((ExtValue == ConvectionConstants::HcExt_ASHRAESimpleCombined) || (ExtValue == ConvectionConstants::HcExt_TarpHcOutside) ||
+                            (ExtValue == ConvectionConstants::HcExt_MoWiTTHcOutside) || (ExtValue == ConvectionConstants::HcExt_DOE2HcOutside) ||
+                            (ExtValue == ConvectionConstants::HcExt_AdaptiveConvectionAlgorithm)) {
```

Comment:
Moving away from string comparisons in lieu of int comparisons. These eventually will become enums.